src/utils/classes/pattern.ts

```diff
@@ -0,0 +1,34 @@
+import { IPatternSettings } from "./models-interfaces";
+
+export class Pattern {
+	// metronome sound should not be here?
+	// private metronomeSound: MetronomeSounds = MetronomeSounds.MetronomeSoundOff;
+
+	private defaultPatternSettings: IPatternSettings = {
+		playNotes: 'quarterNotes',
+		playFillOn: { beat: '3', subBeat: '0' }, // can we make these objects into classes of their own or will it be overkill?
+		timeSignature: { beats: '4', division: '4' }, // can we make these objects into classes of their own or will it be overkill?
```

Comment:
For me it would be overkill to have them as classes. They are clearer as object literals and they will never have more than two properties.